components/accelerator/nvidia/hw-slowdown/component.go

```diff
@@ -152,7 +152,7 @@ func (c *component) States(ctx context.Context) ([]components.State, error) {
 					common.RepairActionTypeHardwareInspection,
 				},
 				Descriptions: []string{
-					"Check the GPU hardware status",
+					"Hardware slowdown are often caused by GPU overheating, power supply unit (PSU) failing or hardware damage, please do a hardware inspection to mitigate the issue",
```

Comment:
what is hardware damage... 

> Constant hardware slowdowns are often the result of either a failing power supply (PSU) or elevated GPU temperatures.

Reply:
like cooling system pin broken..will remove it